simplify language (#198)
nais/doc: docs/naisjob/README.md

# Naisjob

> A Naisjob is very similar to a [NAIS Application](../nais-application/README.md), just that it only runs once.

Reviewer: ```suggestion
A Naisjob is very similar to a [NAIS Application](../nais-application/README.md), except that (by default) it only runs once.
```

> It is built on Kubernetes [Job](https://kubernetes.io/docs/concepts/workloads/controllers/job/) and [CronJob](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/).

Reviewer: ```suggestion
It is built on the Kubernetes [`Job` resource](https://kubernetes.io/docs/concepts/workloads/controllers/job/) and [CronJob](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/).
```

> Compared to an Application, a Job will only start a pod once. 

Reviewer: ```suggestion
Compared to an Application, a `Job` will only start a pod once (unless it fails, usual retry-mechanics hold for `Job`s too).
```

Reviewer 2: Hvis vi skal ha med `unless... ` så syns jeg den burde presis for jobs. Enten lenke til doc, eller ha mer info etter på.

> A CronJob runs Jobs on a time-based schedule, as denoted in `.spec.schedule`.

Reviewer: ```suggestion
A `CronJob` runs `Job`s on a time-based schedule, as denoted in `.spec.schedule`.
```

Reviewer 2: Fikser du de andre cronjob og job som mangler fnutter?

> ```
> 
> !!! warning
>     If your Naisjob is [running in GCP](../clusters/gcp/), or if it is using [Vault sidecar](reference/#vaultsidecar) or [Securelogs](reference/#securelogs).
>     You need to shut down all containers running in the pod for the `Job` to complete.
>     See [Shutting down extra containers](#shutting-down-extra-containers) for more information.
> 
> ## Applying your Naisjob to Kubernetes
> You can deploy your Naisjob just as you would deploy your Application using [NAIS deploy](../deployment/README.md).
> 
> ## Shutting down extra containers
> If your Naisjob has any of these traits:
> 
>  * [runs in GCP](../clusters/gcp/)
>  * [uses Vault sidecar](reference/#vaultsidecar)
>  * [uses Securelogs](reference/#securelogs)
> 
> You will need to shut down all containers running in the pod for the `Job` to complete.

Reviewer: Here I feel as if lines 30-31 (in new version) are repeated (almost verbatim) in more verbose (and easier to read) form in lines 38-44.

Here's my suggested edit:
```suggestion
    Make sure to read the caveat [about shutting down all containers](#shutting-down-extra-containers) for your Naisjob to successfully complete.

## Applying your Naisjob to Kubernetes

You can deploy your Naisjob just as you would deploy your Application using [NAIS deploy](../deployment/README.md).

## Shutting down extra containers

If your Naisjob has any of these traits:

     * [runs in GCP](../clusters/gcp/)
     * [uses Vault sidecar](reference/#vaultsidecar)
     * [uses Securelogs](reference/#securelogs)

You will need to shut down all containers running in the pod for the `Job` to complete.
```
